packages/shared/src/logger.fatal.error.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Utility error to throw expecting that Logger will fatal log its contents
+ */
+export class LoggerFatalError extends Error {
+    obj: object;
```

Comment:
For some reason our linter says to use `Record<string, unknown>` rather than `Object` I dont know what a `object` is in terms of typing, I cant ctrl click it to goto its defs

Not too sure the reasoning. but make it a `Record<>`